pkg/management/endpoint.go

```diff
@@ -68,6 +68,9 @@ type Endpoints struct {
 	DeleteGroup          endpoint.Endpoint
 	GetAuthorizations    endpoint.Endpoint
 	UpdateAuthorizations endpoint.Endpoint
+	PutAuthorization     endpoint.Endpoint
```

Comment:
AddAuthorization

Reply:
Corrected in commit e65d1abb21021efc38ddf4adf3ef93c6273bd283